src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetSemanticInfoTests.cs

```diff
@@ -15213,5 +15213,24 @@ class K
 
             Assert.False(semanticInfo.IsCompileTimeConstant);
         }
+
+        [Fact]
```

Comment:
@brianhartung No problem. For future reference, if you say the test failed without your change we usually take your word on it :)